### PR TITLE
Make project up-to-date

### PR DESCRIPTION
**Pull request by mmBs**
* Remove .idea files
* Bump dependencies 
* Bump Gradle wrapper
* Add google() repository 
* Update travis.yml

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
     defaultConfig {
         applicationId "com.lucasurbas.masterdetail"
         minSdkVersion 21
-        targetSdkVersion 25
+        targetSdkVersion 26
         versionCode commitCount
         versionName "1.1.0"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
         resConfigs "en"
+        vectorDrawables.useSupportLibrary = true
```
Comment (lurbas) [inline app/build.gradle]: Min sdk is 21 so I believe this is not needed

File: build.gradle [hunk 2]
```diff
@@ -3,17 +3,20 @@
...
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
-        // Better IDE support for annotations (so Android Studio interacts better with Dagger)
-        classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta7'
```
Comment (lurbas) [inline build.gradle]: please refer to a stable version, as AS 3.0 is still in beta

**Review by lurbas — CHANGES_REQUESTED** [review]
Thanks for this PR. I know the project is a little dusty right now. One comment. I don't want to introduce gradle plugin in beta version. Which is a bummer if you use AS 3.0. But these beta plugins are deleted after new version is out and the project won't compile. We have to wait for stable 3.0

File: app/build.gradle (hunk 1 above)
Comment (mmBs) [reply app/build.gradle]: Yeah, I forgot to remove it, because I played a little bit with SDK < 21

**Comment by mmBs** [comment]
I reverted to the stable gradle build tools plugin and removed support for vector drawables.

**Review by lurbas — APPROVED** [review]
Thanks for the changes!

**Comment by EypCnn** [comment]
Error:CreateProcess error=2,The system cannot find the file specified

Consult IDE log for more details (Help | Show Log)  


Please Help

**Pull request merged**